### PR TITLE
Add some examples

### PR DESCRIPTION
**Pull request by rtimpe**
These are all llm-generated examples/fixes for a couple of different graph breaks.  I think they're reasonable, but I could use some feedback on how representative they are of real user code (i.e. would they actually be helpful to a user).

### CODE REVIEW

File: docs/gb/gb0206.md [hunk 1]
```diff
@@ -28,6 +28,45 @@ setattr() on Tensor.requires_grad not supported. Mutating requires_grad can intr
...
+    return x * 2
+
+
+x = torch.randn(3)
+x.requires_grad = True  # set outside the compiled region
+fn(x)
+```
+
+If you only need gradients to flow from a specific point onward, use
+`tensor.requires_grad_()` or `tensor.detach().requires_grad_()` outside the
```
Comment (williamwen42) [inline docs/gb/gb0206.md]: Can we get a code example of this?

File: docs/gb/gb0291.md [hunk 2]
```diff
@@ -28,6 +28,107 @@ logging.Logger methods are not supported for non-export cases.
...
+
+You may have seen `torch._dynamo.config.reorderable_logging_functions` used to
+handle `print` and similar calls (it lets Dynamo *reorder* the call to run after
+the compiled graph, so the output is preserved). It does **not** apply to this
+graph break: bound `logging.Logger` methods (`logger.info`, `logger.debug`, ...)
+go through a different code path that only consults `ignore_logging_functions`.
+Adding `logging.Logger.info` (bound or unbound) to
+`reorderable_logging_functions` will not prevent the break.
+
+`reorderable_logging_functions` only works for free functions such as `print`,
```
Comment (williamwen42) [inline docs/gb/gb0291.md:104]: Might have to double check on this one - I'm not sure if we fixed it or if there's another config that could be used for this. Docs on this may be outdated.

File: docs/gb/gb0291.md [hunk 3]
```diff
@@ -28,6 +28,107 @@ logging.Logger methods are not supported for non-export cases.
...
+def fn(x):
+    return x + 1
+
+
+x = torch.randn(3)
+logger.info("processing tensor of shape %s", x.shape)
+fn(x)
+```
+
+If you want to keep the logging call where it is, add the method to
```
Comment (williamwen42) [inline docs/gb/gb0291.md:72]: We could also suggest using `torch.compiler.is_compiling` but this does require more user code changes. `is_compiling` works for more code in general compared to `ignore_logging_functions` but `ignore_logging_functions` doesn't require user code changes.

**Review by williamwen42 — COMMENTED** [review]
I think that overall, these hints are quite helpful and a good starting point!

File: docs/gb/gb0291.md (hunk 2 above)
Comment (rtimpe) [reply docs/gb/gb0291.md:104]: This still graph breaks for me (though my build is a few days old, it could've been fixed very recently)
```
import logging
import torch

logger = logging.getLogger(__name__)

torch._dynamo.config.reorderable_logging_functions.update({logging.info, logger.info})

@torch.compile(fullgraph=True)
def fn(x):
    logger.info("processing tensor of shape %s", x.shape)
    return x + 1


fn(torch.randn(3))
```

I'm not aware of any other config options besides `reorderable_logging_functions` and `ignore_logging_functions`

File: docs/gb/gb0291.md (hunk 3 above)
Comment (rtimpe) [reply docs/gb/gb0291.md:72]: You mean using `is_compiling` to disable the log message?  The config option seems simpler, but that would be a good thing to include in the "escape hatch" skill

File: docs/gb/gb0123.md
Comment (williamwen42) [inline docs/gb/gb0123.md:24]: @tugsbayasgalan can we set this to True by default?

File: docs/gb/gb0291.md (hunk 3 above)
Comment (williamwen42) [reply docs/gb/gb0291.md:72]: Yes, `is_compiling` would disable the log message

**Review by williamwen42 — APPROVED** [review]
